packages/server/lib/controllers/v1/environment/otlp/postSettings.ts

```diff
@@ -0,0 +1,53 @@
+import { z } from 'zod';
```

Comment:
having one endpoint per setting is a bit insane imo
what do you think of taking this opportunity to create a generic endpoint (that only accepts otlp for now)?

Reply:
it is a bit insane. I also think the design is now overwhelming with all those options and I would vote for not only a endpoints refactor but a UI redesign as well. cc @bastienbeurier 
I would prefer to push this otlp export feature to completion though and not go into a refactor at this point.